modules/cluster-logging-upgrading-logging-from-44.adoc

```diff
@@ -0,0 +1,304 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-upgrading.adoc
+
+[id="cluster-logging-updating-logging-from-44_{context}"]
+= Updating cluster logging from 4.4
+
+After updating the {product-title} cluster, you can update cluster logging from 4.4 to 4.6 by changing the subscription for the Elasticsearch Operator and the Cluster Logging Operator.
```

Comment:
Maybe switch the order the operators in this sentence,  "by changing the subscription for the Elasticsearch Operator and the Cluster Logging Operator" to "by changing the subscription for the Cluster Logging Operator and the Elasticsearch Operator". This will be in line with the order you want them to install later on.

Reply:
Great idea. Done.